lib/Clue.js

```diff
@@ -0,0 +1,16 @@
+class Clue {
+  constructor() {
+
+  }
+
+  validateAnswer(currentClue) {
```

Comment:
I worked on this with the Round class today so I can move my method from Round over to the Clue class here.